integration/src/wallets/metamask.ts

```diff
@@ -66,6 +66,11 @@ export function selfTest(get: () => core.HDWallet): void {
     expect(wallet.ethSupportsNativeShapeShift()).toEqual(false);
   });
 
+  it("does not supports bip44 accounts", async () => {
+    if (!wallet) return;
+    expect(wallet.supportsBip44Accounts()).toEqual(false);
```

Comment:
This is a tricky one - when the snap is released, we will be able to provide BIP44 account support through MetaMask for all chains supported by the snap (this includes ETH.) 

We should probably add a conditional return here. If the MetaMask provider has the ShapeShift snap installed and activated, return true. Otherwise, return false. I provide a helper function in the ShapeShift snap adapter that does exactly this.